code/modules/black_market/black_market_item.dm

```diff
@@ -255,85 +305,51 @@ for anyone but the person committing mass murder.
-	item = /obj/item/weapon/gun/portalgun
-	sps_chances = list(0,20,35)
-	stock_min = 1
-	stock_max = 1
-	cost_min = 1800
-	cost_max = 2200
-	display_chance = 80
-
-/datum/black_market_item/animal
-	category = "Living or Once Living Goods."
+/datum/black_market_item/agriculture
+	category = "Agriculture and Animals"
 
-/datum/black_market_item/animal/carp
+/datum/black_market_item/agriculture/carp
 	name = "Space Carp"
-	desc = "One baby space carp.  No refunds."
+	desc = "One baby space carp. No refunds."
 	item = /mob/living/simple_animal/hostile/carp/baby
-	sps_chances = list(50, 60, 70)
 	stock_min = 1
 	stock_max = 10
 	cost_min = 200
 	cost_max = 600
 	display_chance = 80
-
+	
+/datum/black_market_item/agriculture/walkingmushroommycelium
+	name = "Walking Mushroom Seeds"
+	desc = "Sentient mushfriends for all your mushy needs. Be careful though, they bite."
+	item = /obj/item/seeds/walkingmushroommycelium
+	delivery_available = list(1, 1, 1)
+	stock_min = 3
+	stock_max = 3
+	cost_min = 50
+	cost_max = 100
+	display_chance = 99	
+	
 /datum/black_market_item/arcane
 	category = "Supernatural and Arcane Objects"
 
 /datum/black_market_item/arcane/levitation
 	name = "Potion of Levitation"
 	desc = "Potions come in many shapes and sizes, but this one makes you float! Why? Because it looks fucking cool. Maybe you can convince somebody you're the fourth coming of Jesus."
 	item = /obj/item/potion/levitation
-	sps_chances = list(0,0,5)
 	delivery_available = list(0, 1, 1)
 	stock_min = 2
 	stock_max = 4
 	cost_min = 400
 	cost_max = 500
 	display_chance = 70
 
-/datum/black_market_item/arcane/health_potion
-	name = "Potion of Health? Death?"
-	desc = "Unfortunately, some idiot managed to mix together the shipment of identical-looking health potions and death potions. He's dead now. Test out your luck!"
-	item = /obj/item/potion/deception
-	sps_chances = list(0, 10, 30)
-	delivery_available = list(0, 1, 1) //Would shatter on impact
-	stock_min = 2
-	stock_max = 4
-	cost_min = 600
-	cost_max = 800
-	display_chance = 70
-
-/datum/black_market_item/arcane/health_potion/after_spawn(var/obj/spawned, var/mob/user)
-	if(prob(40))
-		spawned = /obj/item/potion/healing
-		
-/datum/black_market_item/plants
-	category = "Seeds" 
-
-/datum/black_market_item/plants/walkingmushroommycelium
-	name = "packet of walking mushroom seeds"
-	desc = "Sentient mushfriends for all your mushy needs"
-	item = /obj/item/seeds/walkingmushroommycelium
-	sps_chances = list(0, 10, 30)
-	delivery_available = list(0, 1, 1)
-	stock_min = 3
-	stock_max = 3
-	cost_min = 50
-	cost_max = 100
-	display_chance = 99	
-
 /datum/black_market_item/toy
 	category = "Recreational and Novelty Items"
 
 /datum/black_market_item/toy/dorkcube
 	name = "Strange Box"
-	desc = "A stolen box filled with unknown loot.  Something is sloshing inside."
+	desc = "A stolen box filled with unknown loot. Something is sloshing inside, wonder what it is..."
```

Comment:
Whoa...... bad writing...........

Reply:
To be actually helpful, I would replace the ellipsis with a question mark.  I also don't like how jank the second half of the now question is tacked on.  "I wonder what it is" would be a lot better, or even "you'll wonder what it is!" to entice someone into buying it.
There is also the fact that this is a change unlisted anywhere in the OP, changelog, and so forth.  You seem to have a habit.

Reply:
I would personally just remove everything after the comma.

Reply:
Forgot about this change, changed it back.